src/Microsoft.Azure.WebJobs.Host/Loggers/FunctionInstanceLogger.cs

```diff
@@ -22,7 +23,7 @@ public FunctionInstanceLogger(ILoggerFactory loggerFactory)
 
         public Task<string> LogFunctionStartedAsync(FunctionStartedMessage message, CancellationToken cancellationToken)
         {
-            string traceMessage = string.Format(CultureInfo.InvariantCulture, "Executing '{0}' (Reason='{1}', Id={2})", message.Function.ShortName, message.FormatReason(), message.FunctionInstanceId);
+            string traceMessage = string.Format(CultureInfo.InvariantCulture, "Executing '{0}' (Reason='{1}', Id={2})", message.Function.LogName, message.FormatReason(), message.FunctionInstanceId);
```

Comment:
Whats the motivation for removing the class name from the log statement? That's useful information - I don't know why we'd want to remove it.

Reply:
User logs on Portal look like this with shortname:
2018-03-08T17:59:53.645 [Information] Executing 'Functions.QueueTriggerCSharp1' (Reason='This function was programmatically called via the host APIs.', Id=84843c45-10d6-44c8-a7b6-aff96f575278)

Changed to using LogName instead.

Reply:
This is the core SDK though, not Functions specific, and in general the disambiguation offered by including the class name is beneficial and was intentional. The Dashboard and other aspects of logging are using ShortName - it's pretty consistent across all core SDK logging/Dashboard infrastructure.

I'd rather not reduce the quality of core SDK logs just because in Functions Portal UI we have natural scoping and the info might be redundant.

Reply:
Please note that this only changes the message in the log statement. Everywhere else  we still use the shortname. Also, application insights has category which is also shortname.